### DISCUSSION TITLE
Need Help! AdGuard Home v0.106.3 is now available! Click here for more info.

### DISCUSSION THREAD

**Discussion by mdziegiel** [AdguardTeam/AdGuardHome#3604] [General]
Hi new to Adguard - I recently just switched from a Pihole to Adgaurd. I have Adguard installed but noticed at the top of the webpage that it says AdGuard Home v0.106.3 is now available! Click here for more info. I Can’t for the life of me find any instruction or a tutorial on how to update to the next version. Could someone point me int he right direction or send me over some instructions. I'm running Adgaurd on a PI 4 and I’m assuming there is some sort of CL for this.

Thanks in advance!

**Comment by ameshkov** [comment]
Hi, update is usually automated unless you're running a non-standard configuration.

Here's the instruction:
https://github.com/AdguardTeam/AdGuardHome/wiki/Getting-Started#update
